login-signup/css/styles.css

```diff
@@ -72,7 +56,7 @@ html {
   align-items: center;
 }
 
-.sso_button-logo {
+.sso__button__logo {
```

Comment:
1. `.sso` isn't a block, we can use `.sso_button` as block class name and `.sso_button__logo` as element class name.
2. Avoid multi-level hireachy in BEM naming convention